packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2679,6 +2680,11 @@ export interface LocalVideoStreamVideoEffectsState {
     isActive: boolean;
 }
 
+// @beta
+export interface LocalVideoTileOptions {
```

Comment:
Should LocalVideoTileSize be moved into here as that is also a LocalVideoTile option

Reply:
Oo this is just for composites